types/okta__okta-vue/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for okta-vue 1.0
+// Project: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue
+// Definitions by: Mike Dodge <https://github.com/innovation-team>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { VueConstructor, PluginFunction } from 'vue';
+import { NavigationGuard } from 'vue-router';
+
+export interface OktaVueOptions {
+    issuer: string;
+    client_id: string;
+    redirect_uri: string;
+    scope: string;
+}
+export function OktaVuePlugin(vm: VueConstructor, options: OktaVueOptions): void;
+export namespace OktaVuePlugin {
```

Comment:
It looks like the plugin is actually an object with an `install` and a `handleCallback`: https://github.com/okta/okta-oidc-js/blob/master/packages/okta-vue/src/Auth.js#L93

Reply:
Making update.